pkgs/by-name/ke/keydb/package.nix

```diff
@@ -89,7 +89,8 @@ stdenv.mkDerivation (finalAttrs: {
       --no-latency \
       --timeout 2000 \
       --clients $NIX_BUILD_CORES \
-      --tags -leaks ${if tlsSupport then "--tls" else ""}
+      --tags -leaks \
+      --skiptest "Active defrag edge case" ${if tlsSupport then "--tls" else ""}
```

Comment:
Nit:

As a reader, it's a bit weird to have the last arg on the same line as the skiptest. I feel like this could be organised better with one arg per line.

Reply:
should be better readable